html/elements/abbr.json

```diff
@@ -0,0 +1,59 @@
+{
+  "html": {
+    "elements": {
+      "abbr": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/HTML/Element/abbr",
+          "support": {
+            "webview_android": {
+              "version_added": true
+            },
+            "chrome": {
+              "version_added": "2"
+            },
+            "chrome_android": {
+              "version_added": true
+            },
+            "edge": {
+              "version_added": true
+            },
+            "edge_mobile": {
+              "version_added": true
+            },
+            "firefox": {
+              "version_added": "1",
+              "notes": "Before Firefox 3.6, this element implemented the <code>HTMLSpanElement</code> instead of the standard <code>HTMLElement</code>."
```

Comment:
I think this reads better with the word "interface". `"Before Firefox 3.6, this element implemented the <code>HTMLSpanElement</code> interface instead of the standard <code>HTMLElement</code> interface."`

Reply:
Fixed.